### DISCUSSION TITLE
No receiver process for my miotas in Firefly

### DISCUSSION THREAD

**Discussion by robyy72** [iotaledger/firefly#5208] [Help & Support] [answered]
Hi guys,
I created a receiver address in Firefly for my IOTA Tokens.
They are sent by Huobi but not arrived at my Firefly / Ledger S now after 48h :/
I was expecting to see a pending receive-process in Firefly but nothing is shown..
Any ideas what I could do ?

I also have trouble connecting my ledger with ledger live and the installed iota app running.
Everything is new and should be up to date.

Best
Robert

**Comment by robyy72** [comment, marked as answer]
Ok, received but I took longer as aspected.
Ledger runs better if I first start ledger live and then plugin the hardware.
Best
Robert

**Comment by MiKeF05** [comment]
The time to process a withdrawal from an exchange depends on the exchange itself. Transfers within the iota network usually take currently max. 10s.
If you use Firefly with a ledger device I think it would be best to close all other apps that are connected to the ledger. Because this can cause connection issues.
